frontend/src/app/libs/common-components/components/button/models/button-color.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonColor {
+	Accept = 'accept',
```

Comment:
It's a good practice to name enum values in uppercase notation.

Reply:
not really necessary here, can be as is